### PR TITLE
style: text inputs colors were not consistent in the user settings page

### LINKED ISSUE #1359: Bug: text inputs colors are not consistent
### Describe the bug

The text inputs have changed and now are not consistent in terms of background color

### Steps to reproduce

Beta vs Prod

image

### Browsers

_No response_

### Additional context (Is this in dev or production?)

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by aryanas159**
## Description

fixes #1359 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
Screenshot 2023-07-15 100400

- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: components/atoms/TextInput/text-input.tsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ const TextInput = ({
             id={id || name || ""}
             placeholder={placeholder || ""}
             className={`flex-1 focus:outline-none  ${
```
Comment (takanome-dev) [inline components/atoms/TextInput/text-input.tsx:64]: LGTM :pizza: 

[nit]: use clsx to make it more readable
```suggestion
               className={clsx(
              "flex-1 focus:outline-none",
              disabled && "bg-light-slate-3 cursor-not-allowed text-light-slate-9",
              classNames
            )}
```

Comment (aryanas159) [reply components/atoms/TextInput/text-input.tsx]: @takanome-dev I have made the changes

**Review by takanome-dev — APPROVED** [review]
Nice! Thanks for the help :+1:

File: components/atoms/TextInput/text-input.tsx [hunk 2]
```diff
@@ -61,9 +61,12 @@ const TextInput = ({
...
             placeholder={placeholder || ""}
-            className={`flex-1 focus:outline-none  ${
-              disabled && "bg-light-slate-3  cursor-not-allowed  text-light-slate-9"
-            } `}
+            className={clsx(
+              "flex-1 focus:outline-none",
+              disabled && "bg-light-slate-3 cursor-not-allowed text-light-slate-9",
+              classNames
+            )}
+
```
Comment (NsdHSO) [inline components/atoms/TextInput/text-input.tsx:69]:
```suggestion
```

**Review by NsdHSO — APPROVED** [review]
LGTM

File: components/atoms/TextInput/text-input.tsx (hunk 2 above)
Comment (aryanas159) [reply components/atoms/TextInput/text-input.tsx:69]: @NsdHSO Is there a change I need to make?

Comment (NsdHSO) [reply components/atoms/TextInput/text-input.tsx:69]: Yes, could you please remove this empty line ??

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**